### DISCUSSION TITLE
Publish Quarto from Jupyter to Quarto-Hub

### DISCUSSION THREAD

**Discussion by Hezel2000** [quarto-dev/quarto-cli#4083] [Q&A] [answered]
I made a Quarto document with jupyter and have an Quarto-Hub account to which I would like to publish this. From the documentation it seems on only .qmd files can be published. But I see now documentation or option to convert my Jupyter file into .qmd – only in .html or .docx.

**Comment by cscheid** [comment, marked as answer]
(do you mean "quarto.pub" instead of "Quarto-Hub"?)

What part of the document are you using to interpret that "only .qmd files can be published"? That's not the case and we'd like to fix it. You should be able to publish ipynb files to quarto.pub without any issue. Is something in particular not working for you?

**Comment by Hezel2000** [comment]
Ah, OK, I didn't see I have to upload the .ipynb file, but instead tried the .html file. It is now working perfectly with .ipynb.
I just heard about Quarto and thought it might be ideal for a recent project to test it. I like it! Thanks!
